pkg/gen/python-templates/tables.py

```diff
@@ -0,0 +1,1140 @@
+_CASING_FORWARD_TABLE = {
```

Comment:
I think we should filter out anything that's not mixed case. I think this will greatly lower the probability of making a mistake, and it makes auditing much easier.

Probably worth lining up the semicolons too if it's not too much work...

Reply:
Also, for stuff that's not generated from mustache, we are just putting it in the sdk directory, rather than having two copies of it.